lib/mnemonic.py

```diff
@@ -89,9 +90,8 @@ def normalize_text(seed):
     return seed
 
 def load_wordlist(filename):
-    path = os.path.join(os.path.dirname(__file__), 'wordlist', filename)
-    with open(path, 'r', encoding='utf-8') as f:
-        s = f.read().strip()
+    data = pkgutil.get_data(__name__, os.path.join('wordlist', filename))
```

Comment:
Interesting.  I presume this is a way to just access files within the package.zip.. otherwise it's 100% compatible with the existing `open` approach?

Reply:
Yes, it's a way of reading files packaged along with the code, without needing to know whether they're deployed as individual files or ZIPs or anything else. It should be fine on all platforms.

Reply:
Hmm.. I merged already but let me double-check this. OSX for instance using PyInstaller does some wonky stuff.  Building now and testing...

Reply:
Yeah no. It's fine.  I was afraid that PyInstaller made a .zip, but put the .json files outside of the .zip or something else wonky.  Not the case. Never mind.. :)